PRChecker/Shared/ViewModels/PRListViewModel.swift

```diff
@@ -10,39 +10,118 @@ import Foundation
 
 class PRListViewModel: ObservableObject {
 
+    @Published var watchedPRList = [NetworkPRResult]()
     @Published var additionalFilters: [String: [Filter]]?
     
     private var subscriptions = Set<AnyCancellable>()
     
+    init() {
+        UserDefaults.standard
+            .publisher(for: \.userList)
+            .sink { userList in
+                guard let userList = userList else { return }
+                self.getWatchedPRList(for: userList)
+            }
+            .store(in: &subscriptions)
+    }
+    
     func getPRList(completion: (() -> Void)? = nil) {
+        // TODO: Convert these to Publishers and make it work with the RefreshableScrollView
+        getMyPRList(completion: completion)
+        
+        if let watchedUsers = UserDefaults.standard.userList {
+            getWatchedPRList(for: watchedUsers)
+        }
+    }
+    
+    func getMyPRList(completion: (() -> Void)? = nil) {
```

Comment:
Maybe we could use `async/await` in next step! (Today's iOS study session was interesting w)

Reply:
I was thinking the same. I like combine but this a async await would work better in our application